molecule/vault/molecule.yml

```diff
@@ -0,0 +1,43 @@
+---
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  # docker + systemd recipe courtesy https://github.com/pilou-
+  - name: vault-host
+    image: xenial
+    command: /sbin/init
+    security_opts: ['seccomp=unconfined']
+    tmpfs: ['/tmp', '/run', '/run/lock']
+    volumes: ['/sys/fs/cgroup:/sys/fs/cgroup:ro']
+    hostname: vault-host
+    capabilities:
+      - IPC_LOCK
+    groups:
+      - vault
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+  env:
+    ANSIBLE_ROLES_PATH: ../../playbooks/roles
+  inventory:
+    links:
+      group_vars: group_vars
+      host_vars: host_vars
+scenario:
+  name: vault
+  test_sequence:
+    - destroy
+    - create
+    - converge
+    - verify
+    - destroy
+verifier:
+  name: testinfra
+  options:
+    v: True
+#    s: True
```

Comment:
Stray comment?

Reply:
I like to keep it around for debugging: it's easier to remove the **#** than type again the **s: True**. But it's from a time when I had trouble remembering how molecule maps options to testinfra :-) I'll remove it.